### PR TITLE
Run Command using positional args

### PR DESCRIPTION
**Pull request by tian2992**
First stage on handling #164

Runs for example: `ocdsdata-cli run canada_montreal uruguay --sample` or `ocdsdata-cli run all`

### CODE REVIEW

**Comment by jpmckinney** [comment]
I split my comment in #164 into #169.

I think it's better for `--all` to be an option instead of a magic argument. Magic tends to lead to errors.

The command should also error if both positional arguments and `--all` are present.

**Comment by tian2992** [comment]
Still needs a fix on status to allow querying status for several sources, Would it be needed for this to merge?

**Comment by jpmckinney** [comment]
Did you see my above comment re: "all" special argument?

**Comment by tian2992** [comment]
@jpmckinney gee, I misunderstood it, sorry.

File: ocdsdata/cli/commands/run.py [hunk 1]
```diff
@@ -39,18 +38,24 @@ def configure_subparser(self, subparser):
...
+                print(selected_source)
+                if selected_source in self.sources:
+                    run.append({'id': selected_source, 'source_class': self.sources[selected_source]})
+                else:
+                    print("We can not find a source that you requested! You requested: %s" % selected_source)
+                    quit(-1)
 
         if not run:
-            print("You have not specified anything to run! Try --run=??? or --runall")
+            print("You have not specified anything to run! Try listing your sources names or special argument all")
```
Comment (jpmckinney) [inline ocdsdata/cli/commands/run.py]: This needs to be updated. There's no "special argument all" anymore.

File: ocdsdata/cli/commands/run.py [hunk 2]
```diff
@@ -39,18 +38,24 @@ def configure_subparser(self, subparser):
...
                 run.append({'id': source_id, 'source_class': source_class})
-        elif args.run:
-            if args.run in self.sources:
-                run.append({'id': args.run, 'source_class': self.sources[args.run]})
-            else:
-                print("We can not find a source that you requested! You requested: %s" % [args.run])
-                quit(-1)
+        elif args.source:
+            for selected_source in args.source:
+                print(selected_source)
```
Comment (jpmckinney) [inline ocdsdata/cli/commands/run.py]: I imagine this print statement should be removed. (We aren't printing sources when `--all` is set).

File: ocdsdata/cli/commands/run.py [hunk 3]
```diff
@@ -39,18 +38,24 @@ def configure_subparser(self, subparser):
     def run_command(self, args):
         run = []
 
-        if args.runall:
+        if args.all and args.source:
+            print("You need to either specifiy a source or use all flag, not both.")
```
Comment (jpmckinney) [inline ocdsdata/cli/commands/run.py]: 'specifiy' typo. Also, always refer to flags as e.g. `--all` to avoid confusion.

File: ocdsdata/cli/commands/clean.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import ocdsdata.cli.commands.base
```
Comment (jpmckinney) [inline ocdsdata/cli/commands/clean.py]: Please make the clean command part of another PR.

File: ocdsdata/cli/commands/run.py (hunk 1 above)
Comment (tian2992) [reply ocdsdata/cli/commands/run.py]: all set

**Comment by odscjames** [comment]
> Still needs a fix on status to allow querying status for several sources, Would it be needed for this to merge?

I think that should be a separate issue so no.
